bower.json

```diff
@@ -21,11 +21,11 @@
     "/tests/"
   ],
   "dependencies": {
-    "polymer": "Polymer/polymer#^1.0.0",
-    "dompurify": "^0.8.0"
+    "polymer": "~2.0.0",
+    "dompurify": "firmfirm/dompurify#v0.8.5"
```

Comment:
Right, this repo should probably use the original dompurify (mine is a fork, and I'm not sure if [PR](https://github.com/cure53/DOMPurify/pull/211) is going to be merged at all). I can reopen the PR by branching my fork so that I can keep my override, or you can just change it after the merge if you're planning to do that.